packages/scandipwa/src/component/CurrencySwitcher/CurrencySwitcher.container.js

```diff
@@ -59,18 +83,31 @@ export class CurrencySwitcherContainer extends DataContainer {
 
     containerProps() {
         const { currencyData } = this.props;
+        const { currency: currencyRates } = this.state;
 
-        return { currencyData };
+        return { currencyData, currencyRates };
     }
 
-    render() {
+    renderCurrencySwitcherComponent() {
+        if (!this.state.currency) {
+            return '';
+        }
+
         return (
             <CurrencySwitcher
               { ...this.containerFunctions }
               { ...this.containerProps() }
             />
         );
     }
+
+    render() {
+        return (
+            <>
+            { this.renderCurrencySwitcherComponent() }
+            </>
+        );
+    }
```

Comment:
This render method is useless.
All this method does is call another method and return its value.
There is no additional logic. So please remove it and rename the `renderCurrencySwitcherComponent` method to `render`.

Reply:
NB: Please don't use fragments if there is only one child.